src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Entities/ICacheableEntityExtensions.cs

```diff
@@ -0,0 +1,27 @@
+namespace OrchardCore.Entities;
+
+public static class ICacheableEntityExtensions
+{
+    public static bool TryGetValue(this ICacheableEntity cache, string key, out object value)
+    {
+        value = cache.Get(key);
+
+        return value != null;
+    }
+
+    public static bool TryGetValue<T>(this ICacheableEntity cache, string key, out T value)
+    {
+        var obj = cache.Get(key);
```

Comment:
You can make use of non generic version instead

Reply:
?

Reply:
Can I add some commit(s) if you don't mind?

Reply:
Sure